### PR TITLE
fix lint

### PR DESCRIPTION
**Pull request by TooTallNate**
Otherwise the app won't boot locally 😕

### CODE REVIEW

File: auto-updater.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 const { autoUpdater } = require('electron');
 const { version } = require('./package');
-const notify = require('./notify');
```
Comment (Morantron) [inline auto-updater.js]: I think you still need to require this module, since it adds listeners to the `app` object.

Comment (rauchg) [inline auto-updater.js]: ugh :\

Comment (rauchg) [inline auto-updater.js]: (if that's the case, that is)

Comment (freebroccolo) [inline auto-updater.js]: You can add '// eslint-disable-line no-unused-vars' to get eslint to ignore it.

Comment (TooTallNate) [inline auto-updater.js]: Updated.

Comment (rauchg) [inline auto-updater.js]: I'd love to continue to investigate whether we really need to have this lingering require or not.

**Comment by mciparelli** [comment]
```js
const { version } = require('./package');
require('./notify');
const ms = require('ms');
```
...

File: auto-updater.js (hunk 1 above)
Comment (mciparelli) [inline auto-updater.js]:
```js
const { version } = require('./package');
require('./notify');
const ms = require('ms');
```
...

**Pull request merged**